backend/prisma/schema.prisma

```diff
@@ -32,8 +50,100 @@ model Session {
 }
 
 model Profile {
-  id     String @id @default(uuid())
-  user   User @relation(fields: [userId], references: [id])
-  userId String @unique @default(uuid())
-  photoUrl String? 
-}
\ No newline at end of file
+  profileID String  @id @default(uuid())
+  user      User    @relation(fields: [userId], references: [UserID])
+  userId    String  @unique
+  photoUrl  String?
+}
+
+model Plan {
+  PlanID           String      @id @default(uuid())
+  description      String
+  startDate        DateTime
+  endDate          DateTime
+  createdDate      DateTime    @default(now())
+  specialEventType EventType
+  contact          Contact?    @relation(fields: [contactID], references: [ContactID])
+  contactID        String?     @unique
+  User             Pledge[]
+  FinalGift        FinalGift?
+}
+
+model Pledge {
+  PledgeID     String        @id @default(uuid())
+  plan         Plan          @relation(fields: [planID], references: [PlanID])
+  planID       String
+  user         User          @relation(fields: [userID], references: [UserID])
+  userID       String
+  inviteStatus InviteStatus?
+  role         Role
+  giftChoice   GiftChoice[]
+}
+
+enum Role {
+  PLAN_LEADER
+  FRIEND
+}
+
+enum InviteStatus {
+  INVITED
+  RESENT_INVITE
+  ACCEPTED
+  DENY
+  NOT_APPLICABLE
+}
+
+model Contact {
+  ContactID          String                    @id @default(uuid())
+  email              String?
+  firstName          String
+  lastName           String?
+  phoneNumber        String?
+  address            String
+  notes              String?
+  source             String
+  owner              User                      @relation(fields: [ownerID], references: [UserID])
+  ownerID            String
+  importantDateEvent ImportantDateEvent[]
+  relationship       UserContactRelationship[]
+  Plan               Plan?
+}
+
+model ImportantDateEvent {
+  ConImportEventID String  @id @default(uuid())
+  contact          Contact @relation(fields: [contactID], references: [ContactID])
+  contactID        String
+}
+
+model UserContactRelationship {
+  UserContactRelationshipID String  @id @default(uuid())
+  contact                   Contact @relation(fields: [contactID], references: [ContactID])
+  contactID                 String
+}
+
+model GiftChoice {
```

Comment:
i think this can just be a `Gift`. Jess will be owning the gift voting and etc so all we're responsible for in this PR right now is to create a table representing all possible gifts, and then be able to link that to a Plan, so we probably don't need a Final Gift table

Reply:
Gifts should live separate from the voting/choice mechanism cuz Gifts will also be used as part of wishlists and marketplaces and etc